README.md

```diff
@@ -69,7 +69,7 @@ jobs:
         
         - uses: azure/docker-login@v1
           with:
-            login-server: contoso.azurecr.io

```

Comment:
Login Server need not be a secret. In case the intent was to make it more clear what does this field stands for, we should add more details as / in the comment section